portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,26 +24,31 @@
 import javax.servlet.http.HttpServletResponse;
 import javax.ws.rs.core.MediaType;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   private Gson gson = new Gson();
-
+  private ArrayList<Comment> comments = new ArrayList<Comment>();
```

Comment:
Suggestion: for now, to make testing easier, include some test comments in the data so that after restart of the server you don't need to add new comments to see how they display.

Reply:
Now I read comments from the datastore, which has some comments.